server/methods/accounts/accounts.app-test.js

```diff
@@ -104,6 +104,7 @@ describe("Account Meteor method ", function () {
       const newAddress = account.profile.addressBook[
         account.profile.addressBook.length - 1];
       delete newAddress._id;
+      delete newAddress.failedValidation;
```

Comment:
Why are we deleting this here?

Reply:
Because it is added to the doc automatically due to a defaultValue. I'm not sure why it wasn't an issue prior to my changes, but I'm pretty sure the defaultValue was always there. If you don't want to delete it, then it could be added to the original address being inserted.

Reply:
No, I was just trying to understand why we were deleting it